Zilon.Core/Zilon.Core/StaticObjectModules/PropDepositModule.cs

```diff
@@ -37,6 +39,9 @@ public class PropDepositModule : IPropDepositModule
         /// <inheritdoc/>
         public string Key { get => nameof(IPropDepositModule); }
 
+        /// <inheritdoc/>
+        public event EventHandler Exhausted;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused event 'Exhausted' or invoke it.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5383584)